exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/QueryResources.java

```diff
@@ -177,6 +180,16 @@ public String getRowsPerPageValues() {
     public String getQueryState() {
       return queryState;
     }
+
+    //Used by results.ftl to indicate autoLimited resultset
```

Comment:
Could you please add a space before the comment: `//Used` -> `// Used` here and in other places?